vega_sim/scenario/constants.py

```diff
@@ -7,4 +7,4 @@ class Network(Enum):
     STAGNET1 = "stagnet1"
     STAGNET2 = "vegawallet-stagnet2"
     STAGNET3 = "vegawallet-stagnet3"
-    FAIRGROUND = "fairground"
+    FAIRGROUND = "vegawallet-fairground"
```

Comment:
If you run `make networks` to make sure you have the latest config files, the network config file for fairground is called `fairground.toml`. 

For some reason stagnet2 and stagnet3 are called `vegawallet-stagnet2.toml` and `vegawallet-stagnet3.toml`.

Trying to run on fairground gives this error:
```
(vega-sim-py3.10) bash-3.2$ python -m devops.run -s ETHUSD -n FAIRGROUND
/Users/caed/vega-market-sim/vega_sim/bin/networks-internal/fairground/vegawallet-fairground.toml
/Users/caed/vega-market-sim/vega_sim/bin/networks/fairground/vegawallet-fairground.toml
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/Users/caed/vega-market-sim/devops/run.py", line 94, in <module>
    main()
  File "/Users/caed/vega-market-sim/devops/run.py", line 80, in main
    with VegaServiceNetwork(
  File "/Users/caed/vega-market-sim/vega_sim/network_service.py", line 260, in __init__
    raise ValueError(
ValueError: ERROR! fairground network config could not be found
(vega-sim-py3.10) bash-3.2$ 
```

tldr - I think this should still be `fairground`